packages/web-components/src/components/va-header-minimal/va-header-minimal.tsx

```diff
@@ -33,9 +33,9 @@ export class VaHeaderMinimal {
   @Prop() subheader?: string;
 
   /**
-   * Disables use of heading tags in the minimal header in favor of `<div>` tags. This is for when a heading level 1 needs to be used elsewhere, as there should only be one heading level 1 per page.
+   * Enables use of heading tags in the minimal header instead of `<div>` tags. This is for when a heading level 1 needs to be used in the header, as there should only be one heading level 1 per page.
    */
-  @Prop() disableHeadings?: boolean = false;
+  @Prop() enableHeadings?: boolean = false;
```

Comment:
Since this is an API change, should this be labeled as a `major` update for the next component-library release? 

I saw that you already update `vets-website` with the the change too! Good catch!